### ISSUE TITLE
Cannot be installed alongside neos/buildessentials 7.1.0 and up

### ISSUE THREAD

**Issue by klfman** [flownative/neos-pixxio#13]
neos/buildessentials requires guzzlehttp/guzzle: ^7.0 
flownative / neos-pixxio requires "guzzlehttp/guzzle": "6.*"

Composer output:
```
  Problem 1
    - Root composer.json requires flownative/neos-pixxio ^2.3 -> satisfiable by flownative/neos-pixxio[2.3.0, 2.3.1].
    - flownative/neos-pixxio[2.3.0, ..., 2.3.1] require guzzlehttp/guzzle 6.* -> found guzzlehttp/guzzle[6.0.0, ..., 6.5.5] but the package is fixed to 7.4.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
 ```

**Comment by sorenmalling** [comment]
The Pimcore asset source from @bwaidelich encounters the same issue (here I try to install the Picore asset source after pixxio

```
composer require wwwision/neos-assetsource-pimcore
Info from https://repo.packagist.org: #StandWithUkraine
Using version ^1.1 for wwwision/neos-assetsource-pimcore
./composer.json has been updated
Running composer update wwwision/neos-assetsource-pimcore
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires wwwision/neos-assetsource-pimcore ^1.1 -> satisfiable by wwwision/neos-assetsource-pimcore[1.1.0].
    - wwwision/neos-assetsource-pimcore 1.1.0 requires guzzlehttp/guzzle ^7.3 -> found guzzlehttp/guzzle[7.3.0, 7.4.0, 7.4.1, 7.4.2] but the package is fixed to 6.5.5 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
```

**Comment by kdambekalns** [comment]
The depdency on `guzzlehttp/guzzle` 7.x was added to the `neos/buildessentials` with https://github.com/neos/BuildEssentials/pull/51 - but AFAICS it will work with 6.x just as well.
